engine/core/src/test/java/org/datacleaner/util/ReflectionUtilsTest.java

```diff
@@ -154,6 +155,20 @@ public void testGetFields() throws Exception {
         assertEquals("a", fields[1].getName());
     }
 
+    private Field[] getNonSyntheticFields(Class clazz) {
```

Comment:
I moved the method getNonSyntheticFields into ReflectionUtils but I think it is needed only in tests. The reason for this is the new maven profile for code-coverage. The JaCoCo library enhances classes with an additional (synthetic) field.